cop4610/makefile

```diff
@@ -0,0 +1,2 @@
+p1: main.c
+  gcc main.c -o p1
```

Comment:
This is the type of stuff I was afraid of. There is no tab character (\t) on the second line, here causing the makefile to not work correctly.